src/Umbraco.Web.UI.Client/src/utility.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * A friendly utility collection to replace AngularJs' ng-functions
+ * If it doesn't exist here, it's probably available as vanilla JS
+ * 
+ * Still carries a dependency on underscore, but if usages of underscore from 
+ * elsewhere in the codebase can instead use these methods, the underscore
+ * dependency will be nicely abstracted and can be removed/swapped later
+ * 
+ * This collection is open to extension...
+ */
+(function (window) {
+
+    /**
+     * Equivalent to angular.noop
+     */
+    const noop = () => { };
+
+    /**
+     * Equivalent to angular.copy
+     * Abstraction of underscore.clone
+     */
+    const copy = val => _.clone(val);
+
+    /**
+     * Equivalent to angular.isArray
+     */
+    const isArray = val => Array.isArray(val) || val instanceof Array;
+
+    /**
+     * Equivalent to angular.equals
+     * Abstraction of underscore.isEqual
+     */
+    const equals = (a, b) => _.isEqual(a, b);
```

Comment:
`angular.equals` and `_.isEqual` seem to have very different implementations, so this part probably requires some more testing.

Reply:
Need to check too how we actually use it - might be more efficient to include a method rather than importing from _